resync

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+declare -A BOTS_MAP=(
+   [gtk-debug]=1
+   [gtk-release]=1
+   [wpe-debug]=1
+   [wpe-release]=1
+)
+
+BOTS=()
+
+usage()
+{
+   local exit_code="$1"
+
+   echo "Usage: resync [bot...]"
+   echo "Resyncs test results"
+   echo ""
+   echo "ARGUMENTS:"
+   echo -e "\tbot\t\t${!BOTS[@]}"
+   echo "OPTIONS:"
+   echo -e "\t-h,--help\tPrints this help"
+
+   exit $exit_code
+}
+
+scan_args()
+{
+   local args=("$@")
+
+   for arg in "${args[@]}"; do
+      if [[ "$arg" == "-h" || "$arg" == "--help" ]]; then
+         usage 0
+      else
+         if [[ ! "${BOTS_MAP[$arg]}" ]]; then
+            echo "Invalid argument: $arg"
+            exit 1
+         fi
+         BOTS+=("$arg")
+      fi
+   done
+}
+
+scan_args "$@"
+
+# Clean.
+git clean -d -f
+git stash
+files=($(git diff --name-only))
+for each in "${files[@]}"; do
+   git checkout "$each"
+done
```

Comment:
I think this is a bit aggressive and may clean source code files that are WIP.
I see it does a git stash before to prevent that, but stash doesn't save untracked files.

I understand the goal is to clean the jsonresults directory so the git pull doesn't conflict, so I think we should focus on only clearing that directory. What about doing something like this?:
```
# Clean jsonresults directort
git clean -d -f jsonresults
git checkout -- jsonresults
```

Reply:
Ok, I think it's a good change. Updated PR.